static/specification.html

```diff
@@ -302,7 +302,7 @@ <h2>
           <dfn>Payment end-point</dfn>:
         </dt>
         <dd>
-          A [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
+          An [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
```

Comment:
See "a URL" is correct as per https://url.spec.whatwg.org
```suggestion
          A [=URL=] to a <a data-cite="SPSP#specification">SPSP end-point</a>
```

Reply:
Yeah, I guess so, I'd stomach it better if it was spelled U.R.L :)